compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -167,6 +167,7 @@ xFlagsSet :: [ GeneralFlag ]
 xFlagsSet = [
    Opt_Haddock
  , Opt_Ticky
+ , Opt_KeepRawTokenStream -- Harvest comments when lexing.
```

Comment:
It isn’t really clear to me why we need this. Presumably for HLint but why does HLint care about comments? The reason should probably also be documented in a comment here.

Reply:
Well, we need to make a call. Right now we don't get comments (I checked that turning this on doesn't have a material performance impact for what its worth). Yes, HLint has a couple of rules relating to comments - they're not that big of a deal but they'll never fire if we never harvest them. Other than that do we want/need them? One thing that springs to mind : didn't we use to display comments in the standard library as hover text and didn't that feature mysteriously stop working at some point?

Reply:
Ah I didn’t realize hlint had warnings for comments. In that case, it sounds absolutely reasonable to turn it on. I would suggest to add a comment (no pun intended) about this so the next person won’t get confused as well.
Regarding the stdlib thing, not sure. I think that was before my time or I missed it.